microservices/the_monkeys_users/internal/models/models.go

```diff
@@ -54,17 +54,19 @@ type TheMonkeysUser struct {
 }
 
 type UserProfileRes struct {
-	ProfileId     string         `protobuf:"bytes,1,opt,name=profile_id,json=profileId,proto3" json:"profile_id,omitempty"`
-	Username      string         `protobuf:"bytes,2,opt,name=username,proto3" json:"username,omitempty"`
-	FirstName     string         `protobuf:"bytes,3,opt,name=first_name,json=firstName,proto3" json:"first_name,omitempty"`
-	LastName      string         `protobuf:"bytes,4,opt,name=last_name,json=lastName,proto3" json:"last_name,omitempty"`
-	DateOfBirth   sql.NullTime   `protobuf:"bytes,5,opt,name=date_of_birth,json=dateOfBirth,proto3" json:"date_of_birth,omitempty"`
-	RoleId        int64          `protobuf:"varint,6,opt,name=role_id,json=roleId,proto3" json:"role_id,omitempty"`
-	Bio           sql.NullString `protobuf:"bytes,7,opt,name=bio,proto3" json:"bio,omitempty"`
-	AvatarUrl     sql.NullString `protobuf:"bytes,8,opt,name=avatar_url,json=avatarUrl,proto3" json:"avatar_url,omitempty"`
-	CreatedAt     sql.NullTime   `protobuf:"bytes,9,opt,name=created_at,json=createdAt,proto3" json:"created_at,omitempty"`
-	UpdatedAt     sql.NullTime   `protobuf:"bytes,10,opt,name=updated_at,json=updatedAt,proto3" json:"updated_at,omitempty"`
-	Address       sql.NullString `protobuf:"bytes,11,opt,name=address,proto3" json:"address,omitempty"`
-	ContactNumber sql.NullInt64  `protobuf:"varint,12,opt,name=contact_number,json=contactNumber,proto3" json:"contact_number,omitempty"`
-	UserStatus    string         `protobuf:"bytes,13,opt,name=user_status,json=userStatus,proto3" json:"user_status,omitempty"`
+	AccountId      string         `protobuf:"bytes,1,opt,name=profile_id,json=profileId,proto3" json:"profile_id,omitempty"`
```

Comment:
Remove all the protobuf, name, proto3 and others, just keep json and omitempty tags